src/transformation/mesh_intersection/mesh_intersection.rs

```diff
@@ -708,6 +707,33 @@ mod tests {
         mesh.to_obj_file(&PathBuf::from("same_test.obj"));
     }
 
+    #[test]
+    fn test_non_origin_pos1_pos2_intersection() {
+        let ball = Ball::new(2f32 as Real).to_trimesh(10, 10);
+        let cuboid = Cuboid::new(Vector3::new(2.0, 1.0, 1.0)).to_trimesh();
+        let mut sphere_mesh = TriMesh::new(ball.0, ball.1);
+        sphere_mesh.set_flags(TriMeshFlags::all()).unwrap();
+        let mut cuboid_mesh = TriMesh::new(cuboid.0, cuboid.1);
+        cuboid_mesh.set_flags(TriMeshFlags::all()).unwrap();
+
+        let res = intersect_meshes(
+            &Isometry::translation(1.0, 0.0, 0.0),
+            &cuboid_mesh,
+            false,
+            &Isometry::translation(2.0, 0.0, 0.0),
+            &sphere_mesh, //.clone().scaled(&Vector3::new(1.001, 1.001, 1.001)),
```

Comment:
That will have to be removed, but I added that because setting both positions to `200;0;0` resulted in a lot of holes in the resulting intersection. I can imagine this is floating point imprecision but it was surprising.

Reply:
Interesting. I think it’s worth opening an issue with the example that generates holes.

Reply:
I opened https://github.com/dimforge/parry/issues/270